src/Bundler.js

```diff
@@ -142,7 +142,11 @@ class Bundler extends EventEmitter {
       contentHash:
         typeof options.contentHash === 'boolean'
           ? options.contentHash
-          : isProduction
+          : isProduction,
+      throwErrors:
+        typeof options.throwErrors === 'boolean'
+          ? options.throwErrors
+          : process.env.NODE_ENV === 'test'
```

Comment:
I think we should default this to `false` in CLI and `true` when using the `Bundler` API.

Reply:
Probably, I'll update it

Reply:
Would this be considered a breaking change?

Reply:
Just something I just thought about, parcel already throws errors in the api using the event handler.
So this might cause issues as the promise of bundle only returns on initial build. What do you think?